src/components/UI/SearchDialogBox/SearchDialogBox.tsx

```diff
@@ -31,6 +32,7 @@ export const SearchDialogBox = (props: any) => {
       <div className={styles.DialogBox}>
         <FormControl fullWidth>
           <AutoComplete
```

Comment:
Is there any reason we're using material autocomplete instead of the AutoComplete component?